CMake/kwiver-flags-gnu.cmake

```diff
@@ -116,3 +116,9 @@ if (KWIVER_CPP_COVERAGE   AND   CMAKE_BUILD_TYPE EQUAL "DEBUG")
   # It seems as though the flag isn't detected alone.
   kwiver_check_compiler_flag(-fprofile-arcs)
 endif ()
+
+# GCC Flag used for stripping binaries of any unused symbols
+# Used for reducing the size of kwiver wheel since pypi has 60Mb constraint on wheel size
+if (SKBUILD)
```

Comment:
You are correct that SKBUILD flag is added by scikit build. When the wheel is build, it is built using [setup.py](https://github.com/Kitware/kwiver/blob/dev/overhaul-python-crall-final-rebase/python/setup.py) and sckit-build helps us in reusing parts of our current build process as show in [the cmake_args](https://github.com/Kitware/kwiver/blob/dev/overhaul-python-crall-final-rebase/python/setup.py#L65). Since the process is not same as our building from source process I was using it to steps that are required only in the wheel building process.